packages/multi-test/src/wasm.rs

```diff
@@ -48,7 +60,7 @@ type QueryClosure<T, E> = Box<dyn Fn(Deps, Env, T) -> Result<Binary, E>>;
 
 /// Wraps the exported functions from a contract and provides the normalized format
 /// Place T4 and E4 at the end, as we just want default placeholders for most contracts that don't have sudo
-pub struct ContractWrapper<T1, T2, T3, E1, E2, E3, C = Empty, T4 = String, E4 = String>
+pub struct ContractWrapper<T1, T2, T3, E1, E2, E3, C = Empty, T4 = Any, E4 = Any>
```

Comment:
Why don't you use `T4 = Empty` and `E4 = String`?

`String` is a great default error type. The only restructions those types have is `ToString`. So requiring `Display` is more than necessary.

Reply:
Hmmm.. that might work and be a simpler solution.
Will try

Reply:
Just approved your PR to that end